mp/src/game/shared/neo/weapons/weapon_neobasecombatweapon.cpp

```diff
@@ -465,42 +465,43 @@ void CNEOBaseCombatWeapon::ItemPreFrame(void)
 }
 
 // Handles lowering the weapon view model when character is sprinting
-void CNEOBaseCombatWeapon::ProcessAnimationEvents(void)
+void CNEOBaseCombatWeapon::ProcessAnimationEvents()
 {
 	CNEO_Player* pOwner = static_cast<CNEO_Player*>(ToBasePlayer(GetOwner()));
 	if (!pOwner)
+	{
 		return;
+	}
+
+	const auto next = [this](const int activity) {
+		SendWeaponAnim(activity);
+		if (GetNeoWepBits() & NEO_WEP_THROWABLE) {
+			return;
+		}
+		constexpr auto nextAttackDelay = 0.2;
+		m_flNextPrimaryAttack = max(gpGlobals->curtime + nextAttackDelay, m_flNextPrimaryAttack);
+		m_flNextSecondaryAttack = m_flNextPrimaryAttack;
+	};
 
-	if (!m_bLowered && (pOwner->IsSprinting()) && !m_bInReload && !m_bRoundBeingChambered)
+	if (!m_bLowered && !m_bInReload && !m_bRoundBeingChambered && pOwner->IsSprinting())
```

Comment:
Interrogative: Are we lowering on ladders still?

If so:
`&& (pOwner->IsSprinting() || bPlayerOnLadder)`

Reply:
Good catch - I had to rebase everything because of merge conflict with master but this should be fixed now